clickhouse/content.md

```diff
@@ -148,6 +152,16 @@ Sometimes you may want to create a user (user named `default` is used by default
 docker run --rm -e CLICKHOUSE_DB=my_database -e CLICKHOUSE_USER=username -e CLICKHOUSE_DEFAULT_ACCESS_MANAGEMENT=1 -e CLICKHOUSE_PASSWORD=password -p 9000:9000/tcp %%IMAGE%%
 ```
 
+#### Managing `default` user
+
+The user `default` has disabled network access by default in the case none of `CLICKHOUSE_USER`, `CLICKHOUSE_PASSWORD`, or `CLICKHOUSE_DEFAULT_ACCESS_MANAGEMENT` are set.
+
+There's a way to make `default` user insecurely available by setting environment variable `CLICKHOUSE_SKIP_USER_SETUP` to 1:
+
+```bash
+docker run --rm -e CLICKHOUSE_SKIP_USER_SETUP=1 -p 9000:9000/tcp %%IMAGE%%
```

Comment:
Doh, sorry, I missed that this was new functionality being added in https://github.com/docker-library/official-images/pull/18387 -- in the future, this is safer to keep interdependent PRs as draft until they're actually safe to merge, especially since having a feature that isn't documented is a better state than having a documented feature that doesn't exist yet. :grimacing:

Reply:
Yeah, sorry, I should mark it as draft as well